File: server/models/note.js
const Sequelize = require('sequelize');
module.exports = function (sequelize) {
  const note = sequelize.define('note', {
    title: {
      type: Sequelize.STRING
    },
    text: {
      type: Sequelize.STRING
    }
  });

  return note;
};

File: server_native/src/models/note.rs
#[derive(Serialize,Deserialize)]
#[serde(rename_all = "camelCase")]
pub struct Note {
  pub id: Option<i64>,
  pub title: Option<String>,
  pub text: Option<String>,
  pub created_at: Option<String>,
  pub updated_at: Option<String>
}